hbase_sql/src/main/java/com/splicemachine/mrio/api/core/HBaseSubregionSplitter.java

```diff
@@ -67,11 +67,15 @@
                                 byte[] stopKey = split.getEndKey();
 
                                 if (LOG.isDebugEnabled()) {
-                                    LOG.debug(String.format("Original split [%s,%s]", CellUtils.toHex(startKey), CellUtils.toHex(stopKey)));
+                                    LOG.debug(String.format("Original split [%s,%s] with scan [%s,%s]",
+                                            CellUtils.toHex(startKey), CellUtils.toHex(stopKey),
+                                            CellUtils.toHex(scanStartRow), CellUtils.toHex(scanStopRow)));
                                 }
 
                                 SpliceMessage.SpliceSplitServiceRequest message = SpliceMessage.SpliceSplitServiceRequest.newBuilder()
-                                        .setBeginKey(ByteString.copyFrom(startKey)).setEndKey(ByteString.copyFrom(stopKey)).build();
+                                        .setBeginKey(ByteString.copyFrom(scanStartRow))
```

Comment:
Isn't there a ZeroCopyLiteralByteString that we can use to wrap the byte array without performing the copy?